mocha/test/organisation.test.js

```diff
@@ -0,0 +1,70 @@
+const httpStatus = require("http-status");
+const APP_URL = process.env.MOCHA_URL;
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+
+let orgApiEndPoint = apiEndPoints.organisationsEndpoint;
+let orgId = "5fc7aa97c0cdcc003855f308";
```

Comment:
We should not use particular ObjectIds. Tests would be very volatile and flaky.
I would check if this endpoint 
​/api​/organisations​/ returns list that is not empty.